LanguageFeatures/Augmentation-libraries/defining_augmentation_A01_t02.dart

```diff
@@ -2,17 +2,17 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion Declarations in any library augmentation or the augmented library
-/// are visible to all of the others, including private ones.
+/// @assertion Augmentation declarations interact with part files mainly in
+/// restrictions on where an augmenting declaration may occur relative to the
+/// declaration it augments.
 ///
 /// @description Checks that declarations in a main library are visible in an
-/// augmentation library
+/// augmentation.
```

Comment:
Following the same idea, we could have this (`suggestion` doesn't work here, apparently):
```
/// @description Checks that declarations in the main library are visible in a
/// part file.
```

Reply:
May be "Checks that declarations in the main library are visible in an augmenting part file."?

Reply:
Same issue: It should work even if that part file doesn't contain any augmenting declarations at all.